.github/workflows/spec-update.yml

```
@@ -100,12 +100,12 @@ jobs:
         continue-on-error: true
       - name: Create PR body file
         run: |-
-          echo 'Update AWS Service Spec packages to latest versions' >> PR.md
+          echo 'Updates the L1 CloudFormation resource definitions with the latest changes from `@aws-cdk/aws-service-spec`' >> PR.md
       - name: Add model changelog to PR body file
         if: steps.diff-db.outputs.diff-result
         run: |-
           echo '' >> PR.md
-          echo '**@aws-cdk/aws-service-spec changes:**' >> PR.md
+          echo '**L1 CloudFormation resource definitions changes:**' >> PR.md
```

```suggestion
          echo '**L1 CloudFormation resource definition changes:**' >> PR.md
```

I'm not a native speaker, but I believe this should be singular